xcube_stac/store.py

```diff
@@ -294,3 +432,103 @@ def _assert_valid_opener_id(cls, opener_id: str):
                 f"Data opener identifier must be "
                 f'{DATASET_OPENER_ID!r}, but got {opener_id!r}'
             )
+
+    def _get_items_nonsearchable_catalog(
+        self,
+        pystac_object: Union[pystac.Catalog, pystac.Collection],
+        recursive: bool = True,
+        **open_params
+    ) -> Iterator[Tuple[pystac.Item, str]]:
```

Comment:
```suggestion
    ) -> Iterator[pystac.Item]:
```

Did I miss something? I don't see how your `yield item` below can yield a tuple.

Reply:
Right, will be changed